### PR TITLE
Common widget GzColor replacement

### PR DESCRIPTION
**Pull request by AzulRadio**
# New feature

* Part of #310

## Summary

Replace implementation for lights (`Lights.qml`) and material color (`Material.qml`) in Component Inspector with GzColor introduced in gz-gui#411

**For future maintainers: selecting color with the Material panel has a bug where the color of the material doesn't change when a color is selected. This bug is not introduced by this common widget.**

## Test it

demo_fortress_lightgif

Draft because of a possible bug in the Gazebo which causes the colors of the lights and material color can only be changed once. 

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by jennuine** [comment]
This PR needs a release from `gz-gui6` but I'm waiting until the plotting PR is ready and merged (ref: https://github.com/gazebosim/gz-sim/pull/1571#issuecomment-1190430491)

**Review by jennuine — APPROVED** [review]
LGTM, just one minor comment

File: src/gui/plugins/component_inspector/Light.qml
Comment (jennuine) [inline src/gui/plugins/component_inspector/Light.qml:353]: Let's keep this consistent with the other sub headers (e.g., `Attenuation")

```suggestion
            text: "      Color"
```

**Review by jennuine — COMMENTED** [review]
Actually, here's a couple more comments regarding spacing

File: src/gui/plugins/component_inspector/Light.qml
Comment (AzulRadio) [reply src/gui/plugins/component_inspector/Light.qml:134]: I thought I could remove one of these components after adding the gzcolor widget but it seems that they are still needed. Removed.

File: src/gui/plugins/component_inspector/Light.qml
Comment (AzulRadio) [reply src/gui/plugins/component_inspector/Light.qml:354]: I think removing the bold font might cause trouble for users to tell between the titles and the contents.

Comment (jennuine) [reply src/gui/plugins/component_inspector/Light.qml:354]: Ok, then can you also bold `Direction` and `Spot features` so that they are all consistent? 

Also, can you add a little spacing so that the plot icons aren't at the very edge of the plugin border?

image

**Pull request merged**
